src/main/java/lesson_3/Task_18.java

```diff
@@ -0,0 +1,20 @@
+/*
+Найти в строке не только запятые, но и другие знаки препинания. Подсчитать
+общее их количество.
+ */
+public class Task_18 {
+    public void CountingPunctuation(String text) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.